### ISSUE TITLE
Shortcuts for other add-ons

### ISSUE THREAD

**Issue by ionm** [wshanks/Zutilo#191]
Is there a way for adding shortcuts to other Zotero add-ons such as mdnotes?

**Comment by wshanks** [comment]
Yes. I am not familiar with mdnotes but there are currently shortcuts for Better BibTeX and ZotFile starting here: https://github.com/wshanks/Zutilo/blob/ce9c007b5a6926702c2a8ac71e44bd07ab67708b/addon/chrome/content/zutilo/keys.js#L455-L464
